scripts/publish.sh

```diff
@@ -7,29 +7,29 @@ set -e
 
 # Brings in: ROOT_DIR, EXAMPLES_DIR, LIB_DIR, CODEGEN_DIR, CONTRIB_DIR, DOC_DIR
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-source $SCRIPT_DIR/config.sh
+source "$SCRIPT_DIR/config.sh"
 
-if ! [ -z "$(git status --porcelain)" ]; then
-  echo "There are uncommited changes! Aborting."
-  exit 1
-fi
+#if ! [ -z "$(git status --porcelain)" ]; then
```

Comment:
I had commented this and the `cargo publish` line out while I was working on top of uncomitted changes so I could test that it all ran and then forgot to properly uncomment them both back... woops how embarrassing!